scalableminds/webknossos
# Added type definitions for react-remarkable

I added type definitions for the `react-remarkable` package. Unfortunately, there were not `@types/react-remarkable` definitions available to I had to write my own.

### Steps to test:
- Run type checker
- CI should be enough

------
(Please delete unneeded items, merge only when none are left open)
